sdk/cognitiveservices/Azure.OpenAI.Inference/src/Azure.OpenAI.Inference.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the Microsoft Azure Cognitive Services OpenAI Inference client library</Description>
+    <AssemblyTitle>Microsoft Azure.OpenAI.Inference client library</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>Microsoft Azure OpenAI Inference</PackageTags>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net47</TargetFrameworks>
```

Comment:
What's the rationale behind `net47`?   The target frameworks for Azure SDK packages are intentionally standardized.   Unless there's strong justification that has been approved by the architects, we shouldn't deviate here.